graylog2-web-interface/src/components/inputs/InputsList.jsx

```diff
@@ -20,6 +22,30 @@ const InputTypesActions = ActionsProvider.getActions('InputTypes');
 const InputsStore = StoreProvider.getStore('Inputs');
 const SingleNodeStore = StoreProvider.getStore('SingleNode');
 
+const InputListRow = styled(Row)`
```

Comment:
I guess we can also remove some of these styles from the global styles:
- https://github.com/Graylog2/graylog2-server/blob/f0bb3dc08c0b1ec64445c6eea15f04351cdd50fd/graylog2-web-interface/src/theme/GlobalThemeStyles.jsx#L215-L221
- https://github.com/Graylog2/graylog2-server/blob/f0bb3dc08c0b1ec64445c6eea15f04351cdd50fd/graylog2-web-interface/src/theme/GlobalThemeStyles.jsx#L355-L367